docs/reference/modules/asset.md

```diff
@@ -237,6 +237,28 @@ node app @apostrophecms/asset:clear-cache
 ```
 </AposCodeBlock>
 
+### `reset`
+The `reset` task provides a more comprehensive cleanup than `clear-cache` by removing all build artifacts, clearing the public folders (without affecting releases), and emptying the cache. This is useful when you need a complete reset of your asset build system or when troubleshooting build-related issues.
```

Comment:
"clearing the public folders" is much too scary, I assume you mean `public/apos-frontend`, please specify